app/scripts/move-popup/dimMovePopup.directive.js

```diff
@@ -45,9 +45,9 @@
     };
   }
 
-  MovePopupController.$inject = ['$rootScope', 'dimStoreService', 'dimItemService', 'ngDialog', '$q', 'toaster'];
+  MovePopupController.$inject = ['$scope', '$rootScope', 'dimStoreService', 'dimItemService', 'ngDialog', '$q', 'toaster'];
```

Comment:
no...I forget to remove the scope there...next commit will remove this reference